### DISCUSSION TITLE
Support Ticket Help!

### DISCUSSION THREAD

**Discussion by Pejayy-05** [community/community#138288] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It's been well over a month since I posted a ticket with the id: 2932519, and I haven't gotten any replies yet, I really need help because it includes my academics, please it's very urgent I didn't know GitHub tickets are slow in processing I really need help

**Comment by joemar25** [comment]
Hey, sorry to hear you’re still waiting on your ticket! Sometimes support can be slow (I encounter it on other platform), especially if they're busy or the issue is a bit complex. Make sure you’ve included all the details in your ticket, like the error messages, what steps you’ve already tried, or anything else that could help them understand the problem better.

Also, try following up directly on the support page where you made the ticket. Sometimes adding a polite comment asking for an update can move things along. It’s tough waiting, but hang in there! Hopefully, they’ll get back to you soon.

**Reply by Pejayy-05** [reply]
I've already tried it so far, still waiting for an update tho 😓 I deployed few of my school projects there so I really need to recover it

**Comment by LiteBrite82** [comment]
Hi @Pejayy-05 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
